Add documentation of 'revokeSessions' parameter to appropriate APIs (#3778)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/users/index.md

This operation will transition the user to the status of `RECOVERY` and the user will not be able to login or initiate a forgot password flow until they complete the reset flow.

> This operation provides an option to delete all the user' sessions.  However, if the request is made in the context of a session owned by the specified user, that session will not be cleared.

Reviewer: ```suggestion
This operation provides an option to delete all the user's sessions.  However, if the request is made in the context of a session owned by the specified user, that session isn't cleared.
```

| --------- | ------------------------------------------------ | ---------- | -------- | -------- | ------- |
| id        | `id` of user                                     | URL        | String   | TRUE     |         |
| sendEmail | Sends reset password email to the user if `true` | Query      | Boolean  | FALSE    | TRUE    |

> | revokeSessions | if true, revoke user sessions               | Query      | Boolean  | FALSE    | FALSE   |

Reviewer: ```suggestion
| revokeSessions | When set to `true`, revokes all user sessions, except for the current session               | Query      | Boolean  | FALSE    | FALSE   |
```

Changes a user's password by validating the user's current password

> This operation provides an option to delete all the sessions of the specified user.  However, if the request is made in the context of a session owned by the specified user, that session will not be cleared.

Reviewer: ```suggestion
This operation provides an option to delete all the sessions of the specified user.  However, if the request is made in the context of a session owned by the specified user, that session isn't cleared.
```

This operation can only be performed on users in `STAGED`, `ACTIVE`, `PASSWORD_EXPIRED`, or `RECOVERY` status that have a valid [password credential](#password-object)

##### Request parameters

| Parameter    | Description                                             | Param Type | DataType                             | Required | Default |
| ------------ | ------------------------------------------------------- | ---------- | ------------------------------------ | -------- |---------|
| id           | `id` of user                                            | URL        | String                               | TRUE     |         |
| strict       | If true, validates against password minimum age policy  | Query      | String                               | FALSE    | FALSE   |
| oldPassword  | Current password for user                               | Body       | [Password object](#password-object)  | TRUE     |         |
| newPassword  | New password for user                                   | Body       | [Password object](#password-object)  | TRUE     |         |

> |revokeSessions | if true, revoke user sessions                          | Body       | boolean                              | FALSE    | FALSE  |

Reviewer 2: nit: space

Author: Fixed.

> | revokeSessions | if true, revoke user sessions                         | Body       | boolean                              | FALSE    | FALSE  |

Reviewer: ```suggestion
| revokeSessions | When set to `true`, revokes all user sessions, except for the current session                         | Body       | boolean                              | FALSE    | FALSE  |
```
